### PR TITLE
Separate terms and conditions for user and service

### PR DESCRIPTION
**Pull request by lucasvanhalst**

### CODE REVIEW

**Comment by RubenMattan** [comment]
Not a fan of how this is done...

Can't we create a new url for legal stuff in case its a service?

**Comment by RubenMattan** [comment]
Also a user shouldn't be able to see terms for a service

**Comment by lucasvanhalst** [comment]
That's the point, there's separate terms of use for services and users 
We just don't have the real document yet

**Comment by RubenMattan** [comment]
Well they aren't seperated... DOCS contains all items

File: src/rogerthat/pages/legal/__init__.py [hunk 1]
```diff
@@ -30,7 +30,8 @@
 LANGUAGES = ["en", "nl"]
 DOC_PRIVACY_POLICY = "privacy-policy"
 DOC_TERMS = "terms-and-conditions"
-DOCS = [DOC_PRIVACY_POLICY, DOC_TERMS]
+DOC_TERMS_USER = "terms-and-conditions-user"
+DOCS = [DOC_PRIVACY_POLICY, DOC_TERMS, DOC_TERMS_USER]
```
Comment (RubenMattan) [inline src/rogerthat/pages/legal/__init__.py]: We can't rename to `terms-and-conditions-user` and terms and conditions should not be in DOCS

**Comment by RubenMattan** [comment]
LGTM

**Pull request merged**